commit b39e8ece931a4b4f64cdf9e75fffd6e82828e471
Author: Mark Pearson <mpearson-lenovo@squebb.ca>
Date:   Thu Oct 24 15:55:21 2024 -0400

    platform/x86: think-lmi: improve check if BIOS account security enabled
    
    Improve determination of whether authentication account is enabled by
    checking if either password or certificate is enabled.
    
    Renamed valid to pwd_enabled for better readability.
    
    Signed-off-by: Mark Pearson <mpearson-lenovo@squebb.ca>
    Link: https://lore.kernel.org/r/20241024195536.6992-1-mpearson-lenovo@squebb.ca
    Reviewed-by: Ilpo Järvinen <ilpo.jarvinen@linux.intel.com>
    Signed-off-by: Ilpo Järvinen <ilpo.jarvinen@linux.intel.com>

diff --git a/drivers/platform/x86/think-lmi.h b/drivers/platform/x86/think-lmi.h
index e1975ffebeb4..4728f40143a3 100644
--- a/drivers/platform/x86/think-lmi.h
+++ b/drivers/platform/x86/think-lmi.h
@@ -65,7 +65,7 @@ struct tlmi_pwdcfg {
 /* password setting details */
 struct tlmi_pwd_setting {
 	struct kobject kobj;
-	bool valid;
+	bool pwd_enabled;
 	char password[TLMI_PWD_BUFSIZE];
 	const char *pwd_type;
 	const char *role;